docs/reference/commandline/dockerd.md

```diff
@@ -17,73 +17,74 @@ keywords: "container, daemon, runtime"
 # daemon
 
 ```markdown
-Usage: dockerd [OPTIONS]
+Usage:	dockerd COMMAND
```

Comment:
That looks like a bug, because the `Use` is actually correct; https://github.com/docker/docker/blob/f3a8886d8890d6797a568ab316baf9400ee2e1e1/cmd/dockerd/docker.go#L35

Looks like this can be if the code _thinks_ there's subcommands defined; https://github.com/docker/docker/blob/fac13e4e0764c551c5452e6f8fbcde45cf67ff12/cli/cobra.go#L102